_apps/workflow/workflow-internal-serial/SingularityApp.workflow_internal-serial

```diff
@@ -0,0 +1,24 @@
+%applabels bioinformatics-samtools
+    MAINTAINER bneron@pasteur.fr
+
```

Comment:
this looks great! A few quick comments:

 - should the `%applabels` be for `workflow-internal-serial` ?
 - same with file name extension? (`SingularityApp.workflow-internal-serial`)